DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -427,6 +427,12 @@ fi
 if [[ $ALIEN_JDL_DISABLECASCADES == 1 ]]; then
   export ARGS_EXTRA_PROCESS_o2_secondary_vertexing_workflow+=" --disable-cascade-finder  "
 fi
+# allow usage of TPC-only in svertexer (default: do not)
+if [[ $ALIEN_JDL_USETPCONLY == 1 ]]; then
+  export CONFIG_EXTRA_PROCESS_o2_secondary_vertexing_workflow+=";svertexer.mExcludeTPCtracks=false"
+else 
+  export CONFIG_EXTRA_PROCESS_o2_secondary_vertexing_workflow+=";svertexer.mExcludeTPCtracks=true"
```

Comment:
@ddobrigk `svertexer.mExcludeTPCtracks` is false by default, I would use your previous version
```
if [[ $ALIEN_JDL_EXCLUDETPCONLY == 1 ]]; then
  export CONFIG_EXTRA_PROCESS_o2_secondary_vertexing_workflow+=";svertexer.mExcludeTPCtracks=true"
fi
```

Reply:
Hi Ruben, the point is @chiarazampolli wanted to have the default operation be *true*, without any additional parameter being set (which would be the case in my previous version). If we want that, we either do as I propose or we simply change the default mExcludeTPCtracks in the SVertexerParams.h to true. Any way you and Chiara prefer is fine with me :-D

Reply:
Well, as I wrote by email, we hope to have TPC tracks used by default (which is already the case on the `svertexer.mExcludeTPCtracks` default value level). If the point of Chiara to minimize amount of JDL variable used by default, then we should define this JDL parameter only when we want to disable them, i.e. your previous version.

Reply:
Hello @shahor02 , @ddobrigk , 

The point is that the the JDL should be able to change things only for testing purposes, so we should leave as default what we want to be our default. This means that for me the default you want in this moment is `svertexer.mExcludeTPCtracks=true"`. 

I will now approve, and then you should decide what you want that becomes our default. The PbPb apass1 and all future reconstructions will run with that without extra JDL variables, which are difficult to maintain. If this means changing the logic, we should do it before Friday.

Cheers,

Chiara

Reply:
Hello @shahor02 , 
I understand, but till Felix is ready, the default should be "true". 
When Felix development is ready, and validated, we should change here.

Reply:
My understanding is that till Felix changes are ready, the default is "true", so I am not changing for testing purposes. 
Anyway, I believe that this is a minor thing...